contracts/compliance/modular/modules/ConditionalTransferModule.sol

```diff
@@ -100,6 +100,10 @@ contract ConditionalTransferModule is AbstractModuleUpgradeable {
      */
     event ApprovalRemoved(address _from, address _to, uint _amount, address _token);
 
+    /// Errors
+
+    error NotApproved();
```

Comment:
could add transfer details, in case the transaction is part of a batch it would help to debug to know which element of the batch is causing an issue.